lib/streamlit/config.py

```diff
@@ -1173,6 +1173,18 @@ def _browser_server_port() -> int:
     """,
 )
 
+_create_theme_options(
+    "codeFontSize",
+    categories=["theme", CustomThemeCategories.SIDEBAR],
+    description="""
+        Sets the font size (in pixels) for code blocks and code text.
+
+        This applies to code blocks (ex: `st.code`), as well as font in `st.json` and `st.help`.
+
+        When unset, the font size will be 14px.
```

Comment:
Maybe to be more correct here we could mention the actual rem value instead of pixels. Since if baseFontSize is used, it won't be 14px anymore.